backend/package.json

```
@@ -0,0 +1,46 @@
+{
+  "name": "backend",
+  "version": "1.0.0",
+  "main": "src/index.ts",
+  "license": "MIT",
+  "scripts": {
+    "start": "ts-node src/index.ts",
+    "build": "tsc",
+    "lint": "eslint . --ext .ts",
```

This will help fixing lint errors much faster.

```suggestion
    "lint": "eslint . --ext .ts",
    "lint:fix": "eslint . --ext .ts --fix",
```